pkg/delete/delete.go

```diff
@@ -83,7 +93,12 @@ func deleteSettingsObject(c dtclient.Client, entries []DeletePointer) []error {
 	errors := make([]error, 0)
 
 	for _, e := range entries {
-		externalID, err := idutils.GenerateExternalID(coordinate.Coordinate{Type: e.Type, ConfigId: e.ConfigId})
+
+		if e.Project == "" {
+			log.Warn("Generating legacy externalID for deletion of %q - this will fail to identify newer Settings object. Consider defining a 'project' for this delete entry.", e)
```

Comment:
```suggestion
			log.Warn("Generating legacy externalID for deletion of %q - this will fail to identify newer Settings objects. Consider defining a 'project' for this delete entry.", e)
```

Reply:
I am not sure about this, but it reads funny. 

If we are talking about a single object, we should say 
```suggestion
			log.Warn("Generating legacy externalID for deletion of %q - this will fail to identify a newer Settings object. Consider defining a 'project' for this delete entry.", e)
```

Reply:
Agreed, "a" is better